.travis.yml

```diff
@@ -32,20 +33,33 @@ jobs:
           mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}'
           | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
     - stage: danger lint and jest
-      script:
-        - "travis_wait docker-compose run -e DANGER_GITHUB_API_TOKEN=$DANGER_GITHUB_API_TOKEN -e TRAVIS=$TRAVIS -e CI=$CI -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run danger && npm run lint && npm run jest'"
-    - stage: wdio
-      script:
-        - "travis_wait docker-compose run test-ci npm run wdio -- --suite suite1"
-    -
-      script:
-        - "travis_wait docker-compose run test-ci npm run wdio -- --suite suite2"
-    -
-      script:
-        - "travis_wait docker-compose run test-ci npm run wdio -- --suite suite3"
-    -
-      script:
-        - "travis_wait docker-compose run test-ci npm run wdio -- --suite suite4"
+      script: "travis_wait docker-compose run -e DANGER_GITHUB_API_TOKEN=$DANGER_GITHUB_API_TOKEN -e TRAVIS=$TRAVIS -e CI=$CI -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci bash -c 'npm run danger && npm run lint && npm run jest'"
+    - stage: compile - default theme
+      script: travis_wait docker-compose run test-ci npm run compile:prod
+    - stage: wdio - default theme
+      name: tiny
+      script: travis_wait docker-compose run test-ci npm run tt-wdio -- --formFactors ['tiny']
+    - script: travis_wait docker-compose run test-ci npm run tt-wdio -- --formFactors ['small']
```

Comment:
This really doesn't matter, but because you are isolating the form factor and theme values for this test run and only running english locales, you could use wdio directly
```
travis_wait docker-compose run -e FORM_FACTOR=tiny test-ci npm run wdio
```

Reply:
Is tt-wdio not used by jenkins internally? I get confused about what is used where.